demo-extended/src/main/java/com/novoda/downloadmanager/demo/extended/NotificationReceiver.java

```diff
@@ -0,0 +1,14 @@
+package com.novoda.downloadmanager.demo.extended;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+public class NotificationReceiver extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        context.startActivity(new Intent(context, MainActivity.class).addFlags(Intent.FLAG_ACTIVITY_NEW_TASK));
```

Comment:
Yes but as the action is hidden now, we could use the demo as an example of how to use this. What do you think?